spec/models/course_spec.rb

```diff
@@ -474,142 +482,4 @@
-      end
-    end
-
-    context "for new courses" do
-      let(:existing_site_status) { create(:site_status, :new, site: existing_site) }
-
-      it "sets the site to new when a new site is added" do
-        expect { subject.add_site!(site: new_site) }.to change { subject.reload.site_statuses.size }.
-          from(1).to(2)
-        expect(new_site_status.status).to eq("new_status")
-      end
-
-      it "keeps the site status as new when an existing site is added" do
-        expect { subject.add_site!(site: existing_site) }.
-          to_not change { existing_site_status.reload.status }.from("new_status")
-      end
-
-      it "removes the site status when an existing site is removed" do
-        expect { subject.remove_site!(site: existing_site) }.to change { subject.reload.site_statuses.size }.
-          from(1).to(0)
-      end
-    end
-
-    context "for suspended courses" do
-      let(:existing_site_status) { create(:site_status, :suspended, site: existing_site) }
-
-      it "sets the site to running when a new site is added" do
-        expect { subject.add_site!(site: new_site) }.to change { subject.reload.site_statuses.size }.
-          from(1).to(2)
-        expect(new_site_status.status).to eq("running")
-      end
-
-      it "sets the site to running when an existing site is added" do
-        expect { subject.add_site!(site: existing_site) }.
-          to change { existing_site_status.reload.status }.from("suspended").to("running")
-      end
-    end
-
-    context "for courses without any training locations" do
-      subject { create(:course, site_statuses: []) }
-
-      it "sets the site to new when a new site is added" do
-        expect { subject.add_site!(site: new_site) }.to change { subject.reload.site_statuses.size }.
-          from(0).to(1)
-        expect(new_site_status.status).to eq("new_status")
-      end
-    end
-
-    context "for mixed courses with new and running locations" do
-      let(:existing_site_status) { create(:site_status, :running, :published, site: existing_site) }
-      #this code will be removed and fixed properly in the next pr
-      let(:another_existing_site) { create(:site, code: 'C', provider: provider) }
-      let(:existing_new_site_status) { create(:site_status, :new, site: another_existing_site) }
-
-      subject { create(:course, site_statuses: [existing_site_status, existing_new_site_status]) }
-
-      it "adds a new site status and sets it to running when a new site is added" do
-        expect { subject.add_site!(site: new_site) }.to change { subject.reload.site_statuses.size }.
-          from(2).to(3)
-        expect(new_site_status.status).to eq("running")
-      end
-
-      it "suspends the site when an existing site is removed" do
-        expect { subject.remove_site!(site: existing_site) }.
-          to change { existing_site_status.reload.status }.from("running").to("suspended")
-      end
-    end
-
-    describe '#accrediting_provider_description' do
```

Comment:
I think this should be in a different block and needs to stay in the tests. It's currently nested when it shouldn't be @misaka

Reply:
Good spot. Thanks!

Reply:
Interesting. I wonder who put it there in the first place ...

![image](https://user-images.githubusercontent.com/15608/59355219-8a47b000-8d1e-11e9-9fd6-04140f995778.png)

-- https://github.com/DFE-Digital/manage-courses-backend/blame/84ea318ac52bfe7620be61fa82349762847ae234/spec/models/course_spec.rb#L563

Reply:
I can only wonder who would approve such a change...

Reply:
Damn! Good spot David. I did wonder why there was a merge conflict with this code...

Reply:
It's back in, resolving